css/wsuwp-plugin-idonate-src.less

```diff
@@ -614,6 +614,16 @@
 			font-size: 16px;
 		}
 	}
+
+	.loading {
+	display: inline-flex;
+	justify-content: flex-end;
+	z-index: 1000;
+	background: rgba( 255, 255, 255, .8 ) 
+				url(../images/loader.gif)
+                95% 50% 
```

Comment:
This should be the set to the same background size and positioning (using the `calc` method) as the arrow-down-gry.png (found in `select.fund` block).

This will make it the same size and positioning as the current down arrow.